python/tskit/trees.py

```diff
@@ -3259,6 +3259,75 @@ def dump_text(
                 )
                 print(row, file=provenances)
 
+    def _repr_html_(self):
+        """
+        Called by jupyter notebooks to render a TreeSequence
+        """
+        table_rows = "".join(
```

Comment:
If we're doing html stuff in the util module, maybe we could drop the implementation of this function in there too? It'll keep the line-count of trees.py down a bit and keep the html stuff in one place.

Reply:
Yep, I've moved it there.